To: vim_dev@googlegroups.com
Subject: Patch 7.3.134
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.134
Problem:    Drag-n-drop doesn't work in KDE Dolphin.
Solution:   Add GDK_ACTION_MOVE flag. (Florian Degner)
Files:	    src/gui_gtk_x11.c


*** ../vim-7.3.133/src/gui_gtk_x11.c	2011-02-25 15:15:43.000000000 +0100
--- src/gui_gtk_x11.c	2011-02-25 17:07:36.000000000 +0100
***************
*** 3094,3100 ****
      gtk_drag_dest_set(gui.drawarea,
  		      GTK_DEST_DEFAULT_ALL,
  		      targets, n_targets,
! 		      GDK_ACTION_COPY);
  }
  
  /*
--- 3094,3100 ----
      gtk_drag_dest_set(gui.drawarea,
  		      GTK_DEST_DEFAULT_ALL,
  		      targets, n_targets,
! 		      GDK_ACTION_COPY | GDK_ACTION_MOVE);
  }
  
  /*
*** ../vim-7.3.133/src/version.c	2011-02-25 16:52:13.000000000 +0100
--- src/version.c	2011-02-25 17:08:35.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     134,
  /**/

-- 
Proverb: A nightingale that forgets the lyrics is a hummingbird.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
